pkg/performanceprofile/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -437,3 +464,92 @@ func renderCrioConfigSnippet(profile *performancev2.PerformanceProfile, src stri
 
 	return crioConfig.Bytes(), nil
 }
+
+// Render out the CPU pinning configuration for CRIO and Kubelet
+// The rendered files will make use of the `reserved` CPUs to generate the configuration files.
+// The template files listed below
+// [./assets/performanceprofile/configs/99-workload-pinning.conf] - CRI-O Config
+// [./assets/performanceprofile/configs/openshift-workload-pinning] - Kubelet Config
+//
+// Note:
+// The CRI-O Config for `cpushares` must be set to zero.
+// Not setting it causes the TOML to not be parsed by CRI-O which is why the template value is set to zero.
+// Further, it should not be configurable through the API, as the Kubelet will inject the correct cpu share annotations according to the pod spec.
+// Carried patches in https://github.com/openshift/kubernetes/pull/706
+func renderCPUPinningConfig(cpuv2 *performancev2.CPU, src string) ([]byte, error) {
```

Comment:
Just a suggestion to separate this from the regular cpu pinning terminology to not confuse readers.

```suggestion
func renderManagementCPUPinningConfig(cpuv2 *performancev2.CPU, src string) ([]byte, error) {
```

Reply:
Sure thing, updated!